### DISCUSSION TITLE
How do I add a quill extension like https://github.com/quill-mention/quill-mention

### DISCUSSION THREAD

**Discussion by xchl** [rawilk/filament-quill#13] [Q&A]
How do I add a quill extension like https://github.com/quill-mention/quill-mention

**Comment by rawilk** [comment]
I haven't tried using extensions myself, so I'm not 100% sure on that. It might be possible with the current component, however you may need to do some source diving to figure it out, depending on how extensions are registered on Quill. If it's not, I'd be open to PR's adding that ability from anyone as well.
